### PR TITLE
Switch to builtin nginx base image

### PR DESCRIPTION
**Pull request by dkess**
This PR removes our custom nginx config and instead opts to use the standard nginx Docker image to serve static files.

I also modified the Python code to not use hardcoded directory names. It instead loads files from the current working directory.

Coupled with ocf/services#13.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,8 @@
-FROM docker.ocf.berkeley.edu/theocf/debian:stretch
+FROM nginx
 
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-        nginx \
-        python3 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y python3
```
Comment (chriskuehl) [inline Dockerfile]: should we upgrade all the deps?

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
 
-COPY www /srv/www
-COPY template.py /srv/
-COPY nginx.conf /srv/
-RUN /srv/template.py
+COPY www /usr/share/nginx/html/
+COPY template.py /tmp
 
-USER nobody
```
Comment (chriskuehl) [inline Dockerfile]: looks like the new base image runs as root? can we change that?

**Comment by dkess** [comment]
Are you saying it's possible that Debian stable would release nginx security patches faster than the nginx repo will? That's surprising to me but if that's the case I don't mind using the `theocf` stretch image and installing nginx manually.

Really the point of this PR was to get rid of our custom `nginx.conf` file which feels unnecessary to me.

**Comment by dkess** [comment]
What if I used `nginx:stable`?

**Comment by chriskuehl** [comment]
> Are you saying it's possible that Debian stable would release nginx security patches faster than the nginx repo will?

yes, i would count on debian to provide patches more reliably quickly

File: Dockerfile [hunk 3]
```diff
@@ -1,17 +1,8 @@
-FROM docker.ocf.berkeley.edu/theocf/debian:stretch
+FROM nginx
 
-RUN apt-get update \
```
Comment (keur) [inline Dockerfile]: i would leave this as it was, and just remove `nginx`

**Comment by dkess** [comment]
Going to close this and look into other ways of simplifying this repo.

**Pull request closed without merging**